### DISCUSSION TITLE
Made with MkDocs

### DISCUSSION THREAD

**Discussion by dagnelies** [mkdocs/mkdocs#3783] [Show and tell]
Hi there,

I just thought it would be nice to share websites made with MkDocs.

So I'll start, here is mine: https://webauthn.passwordless.id/

It's the basic theme, slightly customized. Thanks for all that contributed to this tool. Cheers

**Comment by Andre601** [comment]
I have a lot of websites done with MkDocs and the Material theme. Here are some of my more prominent/updated websites:

- Personal Website: https://andre601.ch
- Website for a plugin I make: https://asl.andre601.ch
- Docs for my Discord Bot (Especially proud of the API docs): https://docs.purrbot.site
